tabsie/server.R

```diff
@@ -80,17 +80,48 @@ shinyServer(
     })
 ####### BUTTON PRESSES #####################
     observeEvent(input$clearTheme, {
-      if (!valAuth) return;#break processing of not authorized.
+      #if (!valAuth) return;#break processing of not authorized.
       updateTextInput(session, "titleField", value = "")
       updateTextInput(session, "xLab", value = "")
       updateTextInput(session, "yLab", value = "")
       updateSliderInput(session, "textSize", value = 15)
       updateSliderInput(session, "xLabRotation", value = 0)
       updateSliderInput(session, "xLabHeight", value = 0)
     })
+######## DIV BOX CONTROL for IDEAS and BUGS #################
+    observe({
+      #if (!valAuth) return();#break processing of not authorized.
+      shinyjs::onclick("toggleIdea",toggle(id="ideaDiv",anim=T))
+      shinyjs::onclick("toggleBug",toggle(id="bugDiv",anim=T))
+    })
+
+    observe({
+      #if (!valAuth) return();#break processing of not authorized.
```

Comment:
Commenting valAuth check out since all it currently does is echo the return function object to a non-existant output. However, them exit properly (or check valAuth properly, depending on which problem is causing this) is for a separate bugfix branch.